recipes/yaml-cpp4rkt/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
```

Comment:
I meant that you probably don't need this file at all, just get rid of it ;)

Reply:
Can I get rid of it once in the feedstock, please? ;)

Reply:
No, this isn't needed so we shouldn't merge it at all.

Reply:
You'll just have another round of builds in the feedstock...

Reply:
> No, this isn't needed so we shouldn't merge it at all.

Fair enough. I've removed it.